main.py

```diff
@@ -1,2 +1,5 @@
+import uvicorn
 
-print("hello, todo")
\ No newline at end of file
+
+if __name__ == "__main__":
+    uvicorn.run("app.main:app", reload=True)
```

Comment:
I'd wrap this into a function and put it somewhere into `cli.py`, only calling it here.